### ISSUE TITLE
Last Menu Item is not added to Primary Menu when child theme is activated

### ISSUE THREAD

**Issue by vytisbulkevicius** [Codeinwp/neve-child#1]
With Neve theme activated **Last Menu Item** appears in the sidebar where you can manage what is rendered as the last menu item: https://prnt.sc/w1r846

If you activate [Neve child theme](https://github.com/Codeinwp/neve-child/archive/master.zip) - this option is no longer available:
https://prnt.sc/w1r8xr

This happens because in file:
`/neve/header-footer-grid/Core/Components/Nav.php`

on **line 193** we check (_Last menu item removed for new users and users who didn't have it set._):
`if ( $old_last_menu_item !== false && ! empty( $old_last_menu_item ) ) `

And this condition is not met so "Last Menu Item" is not added.

I'm not sure why this happens and why `$old_last_menu_item` is not empty or set to false with the child theme.

cc @selul

**Comment by vytisbulkevicius** [comment]
reported https://secure.helpscout.net/conversation/1364323947/257154/

**Comment by selul** [comment]
@vytisbulkevicius last menu item is a legacy setting which we no longer support, it should be used the proper components from the builder.

**Comment by vytisbulkevicius** [comment]
@selul, it makes sense now. I was wondering how did I manage to replicate that in a fresh instance.
I found out that I was able to replicate it because I installed a starter site on that website before.

I tried again on a fresh instance and if I install a starter site **Speed**:
![image](https://user-images.githubusercontent.com/36594177/102019949-deea4200-3d7e-11eb-8b66-1bc3083fc6cc.png)

I'm then getting the Last Menu Item. 
It makes sense as probably this starter site was built with the Last Menu Item in place so **old_last_menu_item** exists.

I'm closing this issue, let me know if we need to register an "issue" to update that starter site (I tried 2 other starter sites and was not able to replicate the same).
